agent/Makefile

```diff
@@ -254,10 +254,17 @@ install-lib:
 # NOTE: we deliberately use `/usr/bin/python3` to make sure that when this is
 # invoked inside a python virtualenv the pip installation won't uninstall the
 # existing pbench module, breaking the environment.
+#
+# During the Python installation, it's going to test to see if the installation
+# directory is on the PYTHONPATH (which it is not...), so put it on the path,
+# just for the installation, based on our prefix and the version of Python.
+# For some reason, this only seems to be a problem on RHEL platforms prior to 9.
+pver = $(shell /usr/bin/python3 --version | sed -En -e 's/^Python (3\.[^.]+).*$$/python\1/p')
```

Comment:
Or avoid the sed with

```
python3 -c 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")'
```

And as for your "educated guess" commentary in the description, the `/lib/python{maj}.{min}/site-packages` path suffix seems to be "well known" and widely assumed if not quite unambiguously documented. 😕

Reply:
I liked your suggestion, but I needed to add `python` to it.  😀   Thanks!

Reply:
Of course you could as easily have added the constant `python` in the text outside the `${pver}` below (and this line's already big enough), but I'm satisfied.